suite-native/settings/src/settingsSlice.ts

```diff
@@ -53,11 +54,20 @@ export const appSettingsSlice = createSlice({
 export const selectFiatCurrencyCode = (state: SettingsSliceRootState) =>
     state.appSettings.fiatCurrencyCode;
 export const selectBitcoinUnits = (state: SettingsSliceRootState) => state.appSettings.bitcoinUnits;
+export const selectAreSatsAmountUnit = (state: SettingsSliceRootState) =>
+    selectBitcoinUnits(state) === PROTO.AmountUnit.SATOSHI;
 export const selectIsOnboardingFinished = (state: SettingsSliceRootState) =>
     state.appSettings.isOnboardingFinished;
 export const selectViewOnlyCancelationTimestamp = (state: SettingsSliceRootState) =>
     state.appSettings.viewOnlyCancelationTimestamp;
 
+export const selectIsValueInSats = (
+    state: SettingsSliceRootState,
+    networkSymbol: NetworkSymbol | null | undefined,
+): boolean => {
+    return networkSymbol === 'btc' && selectAreSatsAmountUnit(state);
```

Comment:
nit: extra `{ }` and return
```suggestion
    => networkSymbol === 'btc' && selectAreSatsAmountUnit(state);
```

Reply:
Thanks, I like!